mkdocs.yml

```diff
@@ -56,6 +55,10 @@ nav:
   - 'Code execution': 'codeExecution.md'
   - 'Docker packaging': 'docker.md'
   - 'Contributing to PL development': 'contributing.md'
+- Running in Production:
+  - 'Running in production': 'running-in-production.md'
```

Comment:
Let's call this page "Setup". Could you also move these and the other pages to a `docs/running-in-production` directory to keep them separated from the rest of the docs?

Reply:
Moved to directory and renamed to setup